front/lib/plans/stripe.ts

```diff
@@ -1,14 +1,20 @@
 import type { Result, WorkspaceType } from "@dust-tt/types";
 import type { SubscriptionType } from "@dust-tt/types";
-import { assertNever, Err, Ok } from "@dust-tt/types";
+import { Err, Ok } from "@dust-tt/types";
```

Comment:
Changes in this file are basically just removing the "planCode" param and make it possible to subscribe only to PRO_PLAN_SEAT_29. We may revisit this in the future if we start having several pro plans, but right now we don't and this is simpler 👍